user_guide_src/source/installation/installing_composer.rst

```diff
@@ -82,7 +82,25 @@ Whenever there is a new release, then from the command line in your project root
 
     composer update
 
-Read the :doc:`upgrade instructions <upgrading>`, and check Breaking Changes and Enhancements.
+Read the :doc:`upgrade instructions <upgrading>` and :doc:`change log <../changelogs/index>`,
+and check Breaking Changes and Enhancements.
+
+Upgrading to a Specified Version
```

Comment:
I don't know if my impression is correct or not, but we usually use Upgrading for forward(new ver).

It would be better if you added how to install the special version in the installation section. Similar to what **michalsn**  explained in slack:
```console
composer create-project codeigniter4/appstarter:4.4.8 ci4.4.8
```

Reply:
Added.